src/service/extension-location.js

```diff
@@ -98,9 +98,7 @@ export function calculateEntryPointScriptUrl(
  * @return {!ExtensionInfoDef}
  */
 export function parseExtensionUrl(scriptUrl) {
-  const regex = /^(.*)\/(.*)-([0-9.]+)\.js$/i;
-  const matches = scriptUrl.match(regex);
-
+  const matches = scriptUrl.match(/^(.*)\/(.*)-([0-9.]+|latest)\.js$/i);
```

Comment:
Nit: We can drop the `^(.*)\/` entirely

Reply:
Don't we at least need the `\/` to separate the extension ID? E.g. `https://cdn.ampproject.org/v0/amp-foo-latest.js`.